torch/_utils.py

```diff
@@ -141,9 +141,10 @@ def _rebuild_tensor(storage, storage_offset, size, stride):
     return t.set_(storage._untyped(), storage_offset, size, stride)
 
 
-def _rebuild_tensor_v2(storage, storage_offset, size, stride, requires_grad, backward_hooks):
+def _rebuild_tensor_v2(storage, storage_offset, size, stride, requires_grad, grad, backward_hooks):
```

Comment:
I'm afraid you can't do this for BC reasons.
Indeed, this function will also be called for models that were saved with older versions of pytorch and that didn't serialize the grad argument.

You will have to create a `_rebuild_tensor_v3` function and keep this one's signature as-is (code should be refactored to avoid duplication for the impl of course).

Reply:
Thanks, this would explain why some of the tests complain about `_rebuild_tensor_v2`  and `_rebuild_qtensor` being passed the wrong the number of arguments.